exercises/static/exercises/drone_cat_mouse/web-template/mouse.py

```diff
@@ -20,50 +20,50 @@ def initRobot(cls):
         new_instance = cls()
         return new_instance
 
-    def takeoff(self):
```

Comment:
undo all changes in this file when drone_wrapper backend is ready

Reply:
sure